pbx/model.proto

```diff
@@ -261,6 +262,13 @@ message ClientNote {
 	int32 seq_id = 3;
 }
 
+message ClientExtra {
+	repeated string attachments = 1;
```

Comment:
Perhaps make it a key-value dict to take care of other potential situations in the future?

Reply:
I considered it and decided against k-v. The reason is these values must be understood by the server. There is no point in sending unknown key in Extra if server does not know how to handle it. And if server can handle it then it can be a fixed field in a struct.